execution_workflows/APAlyzer/workflow/scripts/APAlyzer_postprocessing.R

```diff
@@ -0,0 +1,98 @@
+#------------------------Postprocessing------------------------
+# Takes APAlyzer output and generates differential challenge
+# output tsv file
+
+# load libraries
+if ( suppressWarnings(suppressPackageStartupMessages(require("optparse"))) == FALSE ) { stop("[ERROR] Package 'optparse' required! Aborted.") }
+if ( suppressWarnings(suppressPackageStartupMessages(require("hash"))) == FALSE ) { stop("[ERROR] Package 'hash' required! Aborted.") }
+
+#########################
+###  PARSE ARGUMENTS  ###
+#########################
+
+# Get script name
+script = sub("--file=", "", basename(commandArgs(trailingOnly = FALSE)[4]))
+
+# Build description message
+description = "Build reference genome\n"
+version = "Version: 1.0.0 (May 2021)"
+requirements = "Requires: optparse, APAlyzer"
+msg = paste(description, version, requirements, sep = "\n")
+
+# Define list of arguments
+option_list = list(
+  make_option(
+    "--in_postprocessing",
+    action = "store",
+    type = "character",
+    default = FALSE,
+    help = "APAlyzer output variables to preprocess.",
+    metavar = "files"
+  ),
+  make_option(
+    "--out_postprocessing",
+    action = "store",
+    type = "character",
+    default = FALSE,
+    help = "Path for the final output file.",
+    metavar = "files"
+  ),
+  make_option(
+    c("-h", "--help"),
+    action = "store_true",
+    default = FALSE,
+    help = "Show this information and die."
+  ),
+  make_option(
+    c("-v", "--verbose"),
+    action = "store_true",
+    default = FALSE,
+    help = "Print log messages to STDOUT."
+  )
+)
+
+# Parse command-line arguments
+opt_parser = OptionParser(
+                usage=paste("Usage:", script, "[OPTIONS] \n", sep=" "),
+                option_list = option_list,
+                add_help_option = FALSE,
+                description = msg)
+opt = parse_args(opt_parser)
+
+#########################
+###  GENERATE OUTPUT  ###
+#########################
+
+# Load variables from preprocessing step
+load(opt$in_postprocessing)
+out_df = NULL
```

Comment:
This line shouldn't be here. The variable that is loaded in the prior line contains a dataframe with the results from main. This here overwrites it and that's why the file "differential_challenge_output.tsv" is empty.

Reply:
ah yes! omg i added this to test but forgot to remove, thanks for catching this!!